README.md

```diff
@@ -0,0 +1,59 @@
+# Operator Framework for Charms
+
+This framework is not yet stable and is subject to change, but is available
+for early testing.
+
+## Getting Started
+
+Start by creating a charm directory with at least the following files:
+
+* `lib/charm.py` (must be executable and use Python 3.6+)
+* `hooks/install` (or `hooks/start` for K8s charms) sym-linked to `../lib/charm.py`
+* `metadata.yaml`
+
+Then install the framework into the `lib/` directory using:
+
+```
+pip install -t lib/ https://github.com/canonical/operator
+```
+
+Your `lib/charm.py` is the entry point for your charm logic. At a minimum, it
+needs to define a subclass of `CharmBase` and pass that into the framework's
+`main` function:
+
+```python
+from ops.charm import CharmBase
```

Comment:
Using relative imports here, e.g. `from .ops.charm import CharmBase`, is more friendly to PyCharm.

Reply:
This should not be a relative import. They happen to be in the same directory right now, but ops is an independent package and can be anywhere else in the system. We should actually probably reorganize things so that the relative import breaks down.